packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/nodejs/sdkforyourapp.md

```diff
@@ -1,41 +1,12 @@
-## TO DO for NodeJS
 
-#### Step 1: Set up your app for .Net 4.8 or greater
+#### Step 1: Install the SDK into your project application
 
-The SDK and samples are built using .Net 4.8.
+Download and install the SDK into your Express application. See [Building the SDK](https://github.com/okta/okta-auth-js#building-the-sdk) for further information.
 
-#### Step 2: Add the Okta SDK Nuget Packages
+#### Step 2: Install the module dependencies
 
-Before using the SDK in your own app, you need to add the following
-Nuget packages to your project:
+Before using the SDK in your own app, you need to add the node dependencies listed in the `package.json` file.
 
-* `Okta.Idx.Sdk`
-* `Okta.Sdk.Abstractions`
-
-> **Note:** Nuget packages are pre-release. When you search for Nuget
-packages in Visual Studio, ensure that the pre-release checkbox is selected.
-
-#### Step 3: Initialize IdxClient
-
-All functionality in the SDK is accessed through the methods of the
-`IdxClient` object. After you add the Nuget packages, the next step
-is to initialize the `IdxClient`. There are two steps to initialize
-the client:
-
-1. Add the following namespaces:
-   * `Okta.Idx.Sdk`
-   * `Okta.Sdk.Abstractions`
-1. Initialize the `IdxClient`
-
-See the following sample code for more details:
-
-```csharp
-using Okta.Idx.Sdk;
-using Okta.Sdk.Abstractions;
-
-var idxAuthClient = new IdxClient();
+```console
+npm install
```

Comment:
not need, as it's the devs own app, they should have control of the app's dependencies, we probably only need to expose the sdk installation detail.